packages/server/src/workers/reindex.ts

```diff
@@ -219,24 +236,34 @@ function formatResults(results: ReindexJobData['results']): Parameters {
     resourceType: 'Parameters',
     parameter: Object.keys(results).map((resourceType) => {
       const result = results[resourceType];
-      if (typeof result === 'string') {
+      if ('err' in result && result.err) {
```

Comment:
is the `'err' in result` check necessary?  seems redundant?  might be necessary to keep TS happy?

Reply:
Yeah, this was to placate tsc